test/annexB/language/comments/single-line-html-close-unicode-separators.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-html-like-comments
+description: SingleLineHTMLCloseComment
+info: |
+    Comment ::
+      MultiLineComment
+      SingleLineComment
+      SingleLineHTMLOpenComment
+      SingleLineHTMLCloseComment
+      SingleLineDelimitedComment
+
+    SingleLineHTMLCloseComment ::
+      LineTerminatorSequenceHTMLCloseComment
+
+    HTMLCloseComment ::
+      WhiteSpaceSequence[opt] SingleLineDelimitedCommentSequence[opt] --> SingleLineCommentChars[opt]
+negative:
+  phase: runtime
+  type: Test262Error
```

Comment:
I don't think we need the negative tags here.

In this case you may remove this tag and replace the `if` at the end with an `assert.sameValue(counter, 2);`

The assertion api will throw a Test262Error for any failure.